runtime/js/40_process.js

```diff
@@ -144,11 +146,6 @@ function run({
       ...new SafeArrayIterator(ArrayPrototypeSlice(cmd, 1)),
     ];
   }
-  internals.warnOnDeprecatedApi(
-    "Deno.run()",
-    (new Error()).stack,
-    `Use "Deno.Command()" API instead.`,
-  );
```

Comment:
I think we want to keep outputting a warning here, but just update the message.

Reply:
We don't output these warnings anyway - it was disabled shortly after 1.43 release.